flake/modules/home-manager.nix

```
@@ -1,5 +1,9 @@
 # Home Manager module
-packages: lib: {
+{
+  packages,
+  lib,
+  inputs,
```

```suggestion
  self
  lib,
```

Then get `packages` and `inputs` from `self`.